transaction/examples/resources/mint_fungible.rtm

```diff
@@ -0,0 +1,19 @@
+# This transaction manifest shows how a mintable fungible resource may be minted through the 
```

Comment:
It would be nice to pair this with a resource which is mintable via an auth rule - and then this would show proof from the account to do the minting? :)

As I imagine minting with auth will be the main way of minting?

Reply:
Yeah you are right. I will add another example now of how somebody can mint a restricted minting fungible and non-fungible resources :)